OloEngine/src/OloEngine/Scene/Scene.cpp

```
@@ -308,6 +314,104 @@ namespace OloEngine
         OnPhysics3DStop();
     }
 
+    // Process sub-emitter triggers that reference child systems.
+    // For triggers with a valid ChildSystemIndex, emit into the corresponding child system's pool.
+    static void ProcessChildSubEmitters(ParticleSystemComponent& psc, f32 dt, const glm::vec3& emitterPos)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'ProcessChildSubEmitters' of similar type are easily swapped by mistake

(bugprone-easily-swappable-parameters)